conf/utils/README.md

```diff
@@ -0,0 +1,94 @@
+## Create Welsh messages file
+
+### create-welsh-translations.py
+
+Given a spreadsheet with columns
+
+| CDRS Messages                    |                          |                                    |
+|----------------------------------|:-------------------------|:-----------------------------------|
+|Message key                       | Message value            | Welsh Translation of message value |
+|#                                 |                          |                                    |
+|#  GLOBAL MESSAGES                |                          |                                    |
+|#                                 |                          |                                    |
+|generic.day                       | Day                      | Diwrnod                            |
+|generic.month                     | Month                    | Mis                                |
+|generic.year                      | Year                     | Blwyddyn                           |
+|generic.yes                       | Yes                      | Iawn                               |
+
+The script converts them into a file containing **key** and **Welsh translation** as follows
+```
+#===================================================
+#  GLOBAL MESSAGES
+#===================================================
+generic.day=Diwrnod
+generic.month=Mis
+generic.year=Blwyddyn
+generic.yes=Iawn
+```
+To use a script run command
+```
+python create-welsh-translations.py test.csv
+```
+where `test.csv` is the input CSV file name including path.   
+ 
+Default outcome file name is `messages.cy`. If you wish to change it, you can specify it in the command line arguments.
+```
+python create_welsh_translations.py test.csv my_outcome_file_name.txt
+```
+Output
+```
+% » python create-welsh-translations.py translations.csv
+Finished
+Invalid records:  2
+* ['Upload files page', '', '', '', '']
+* ['Wait a few seconds and then select ‘continue’', '', '', '', '']
+```
+
+### create-welsh-messages-no-keys.py
+Given a spreadsheet with columns
```

Comment:
I think you should add a comment clarifying that this is not the way that we want to receive files and that this script is only there in case you do it by mistake.